lib/kafo/kafo_configure.rb

```diff
@@ -28,12 +28,15 @@ module ENV
 require 'kafo/scenario_manager'
 require 'kafo/execution_environment'
 require 'kafo/logging'
+require 'kafo/app_option/declaration'
 
 module Kafo
   class KafoConfigure < Clamp::Command
     include StringHelper
 
     class << self
+      include AppOption::Declaration
+
       attr_accessor :config, :root_dir, :config_file, :gem_root,
                     :module_dirs, :kafo_modules_dir, :verbose, :app_options, :logger,
```

Comment:
You should drop `app_options` here as well.
```suggestion
                    :module_dirs, :kafo_modules_dir, :verbose, :logger,
```